CONTRIBUTING.md

```diff
@@ -7,7 +7,7 @@
 
 # Contributing
 
-We welcome suggested improvements and bug fixes for `sentry-cocoa`, in the form of pull requests on [`GitHub`](https://github.com/getsentry/sentry-javascript). The guide below will help you get started, but if you have further questions, please feel free to reach out on [Discord](https://discord.gg/Ww9hbqr).
+We welcome suggested improvements and bug fixes for `sentry-cocoa`, in the form of pull requests on [`GitHub`](https://github.com/getsentry/sentry-cocoa). Please follow our official [Commit Guidelines](https://develop.sentry.dev/code-review/#commit-guidelines). The guide below will help you get started, but if you have further questions, please feel free to reach out on [Discord](https://discord.gg/Ww9hbqr).
```

Comment:
L: this link leads to https://github.com/orgs/getsentry/teams/cocoa which is broken

Reply:
Sorry @marandaneto I don' get your point. The discord link works for me.

Reply:
open this link: https://develop.sentry.dev/code-review/#commit-guidelines
navigate to the teams section: https://develop.sentry.dev/code-review/#github-teams
click on `@getsentry/cocoa` which leads to this URL https://github.com/orgs/getsentry/teams/cocoa and its broken aka 404, it's not totally related to this PR, but either the GH link or the team should exist under the Sentry org on GH